virttest/xml_utils.py

```diff
@@ -84,9 +86,90 @@ def unlink(self):
         """
         try:
             os.unlink(self.name)
-            self.close()
+            self.open_file.close()
```

Comment:
self.close() is ok, you have written the function

Reply:
updated!